plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcConnectorTest.java

```diff
@@ -142,6 +143,39 @@ public void testTopNPushdown()
                 .isFullyPushedDown();
     }
 
+    @Test
+    public void testNullSensitiveTopNPushdown()
```

Comment:
nit: test could be a separate commit

Reply:
It could be. OTOH, before SQL Server TopN is added, the test was not needed, as the only TopN implementation (PostgreSQL), supports NULL FIRST, NULL LAST syntax directly, so there was no room for error.
Do you think it could stay here?

Reply:
"nit" implies "it can"